src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Publisher.kt

```diff
@@ -53,11 +56,12 @@ class Publisher(private val topicName: String) {
         ApiFutures.addCallback(future, object : ApiFutureCallback<String> {
 
             override fun onFailure(throwable: Throwable) {
-                println(throwable)
+                LOGGER.error("Failed on publish with message:", throwable.message, throwable)
```

Comment:
If you want a message to be shown, add a placeholder to the string. In SLF4J placeholder is just a pair of curly braces: 
`"with message: {}"

Reply:
Done